spec/lcm/integration/project_xerox_helper_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'support/configuration_helper'
+require_relative 'support/connection_helper'
+require_relative 'shared_examples_for_synchronization_bricks'
+require_relative 'support/project_helper'
+
+describe GoodData::Helpers::ProjectXeroxHelper do
+  before(:all) do
+    config = LcmConnectionHelper.environment
+
+    source_client = GoodData.connect(
```

Comment:
warning: Similar code found in project_xerox_helper_spec.rb:10, project_xerox_helper_spec.rb:16 (mass = 34) (flay)